client/dive-common/components/MultiCamTools.vue

```diff
@@ -0,0 +1,202 @@
+<script lang="ts">
+import { computed, defineComponent } from '@vue/composition-api';
+import {
+  useCamMap, useEnabledTracks, useHandler, useSelectedCamera,
+  useSelectedTrackId, useTime,
+} from 'vue-media-annotator/provides';
+
+import TooltipBtn from 'vue-media-annotator/components/TooltipButton.vue';
+import {
+  getTrack, getTrackAll,
+} from 'vue-media-annotator/use/useTrackStore';
+import { TrackId } from 'vue-media-annotator/track';
+
+interface CameraTrackData {
+    trackExists: boolean;
+    annotationExists: boolean;
+}
+
+export default defineComponent({
+
+  name: 'MultiCamTools',
+  description: 'MultiCam Tools',
+
+  components: { TooltipBtn },
+
+  setup() {
+    const selectedCamera = useSelectedCamera();
+    const enabledTracksRef = useEnabledTracks();
+    const handler = useHandler();
+    const { frame } = useTime();
+    const selectedTrackId = useSelectedTrackId();
+    const camMap = useCamMap();
+    const cameras = computed(() => [...camMap.keys()]);
+    const tracks = computed(() => {
+      const trackKeyPair: Record<string, CameraTrackData> = {};
+      // EnabledTracksRef causes computed to update on various changes to enusre true reactivity
```

Comment:
I'll update the comment.  Using the other values aren't enough because creating/removing a track from a camera won't update SelectedCamera or SelectedtrackId if there are remaining items.  EnabledTracksRef is based on the merge across tracks so the actions in the multiCam will be updated whenever there is a change.

Reply:
Going to correct myself, I think this only works in the case of linking/unlinking tracks.  I need some other indicator for deletion of a detection/track from a camera.